contracts/Core.sol

```diff
@@ -91,13 +91,17 @@ contract Core is CoreInterface, Util {
 	 * @param _chainIdOrigin origin chain id
 	 * @param _chainIdRemote remote chain id
 	 * @param _openSTRemote remote openSTUtility/openSTValue contract address
+	 * @param _blockHeight block height
+	 * @param _stateRoot state root
 	 */
 	constructor(
 		address _registrar,
 		uint256 _chainIdOrigin,
 		uint256 _chainIdRemote,
 		address _openSTRemote,
-		WorkersInterface _workers)
+		WorkersInterface _workers,
```

Comment:
Could you please, move this as a last argument for constructor. It's logically fits at the end.

Reply:
Done